bevy_renet/src/lib.rs

```diff
@@ -56,11 +85,33 @@ impl RenetServerPlugin {
         }
     }
 
-    pub fn send_packets_system(mut server: ResMut<RenetServer>, mut renet_error: EventWriter<RenetError>) {
+    pub fn send_packets(mut server: ResMut<RenetServer>, mut renet_error: EventWriter<RenetError>) {
         if let Err(e) = server.send_packets() {
             renet_error.send(RenetError::IO(e));
         }
     }
+
+    pub fn with_default_system_setup(mut self, default_system_setup: bool) -> Self {
+        self.default_system_setup = default_system_setup;
+        self
+    }
+
+    pub fn get_systems(label: RenetSystems) -> SystemSet {
+        match label {
+            RenetSystems::ClearEvents => SystemSet::new()
+                .with_system(Events::<ServerEvent>::update_system)
+                .with_system(Events::<RenetError>::update_system)
+                .label(RenetSystems::ClearEvents),
+            RenetSystems::Update => SystemSet::new()
+                .with_system(Self::update_system.with_run_criteria(has_resource::<RenetServer>))
+                .label(RenetSystems::Update),
+            RenetSystems::SendPackets => SystemSet::new().with_system(
+                Self::send_packets
+                    .with_run_criteria(has_resource::<RenetServer>)
+                    .label(RenetSystems::SendPackets),
+            ),
+        }
+    }
 }
```

Comment:
I feel that adding systems like this adds too much noise to the code without gains, we have very few system, the labels almost map 1 to 1. I think we still could use the old way, WDYT?

Reply:
Ya I was just thinking it might be useful if more systems are added in the future.

Reply:
When the time comes, we can use this approach if necessary :+1: 
For now, I would just use after/before the system names and remove the `get_system`.